.vscode/settings.json

```diff
@@ -0,0 +1,56 @@
+﻿{
```

Comment:
Please remove this file from the PR. IDE config files should be ignored.

Reply:
Removed the File.